app/services/search/bing_search_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Search
+  class BingSearchService
+    BING_SEARCH_URL = 'https://www.bing.com/search?q=%{keyword}&setlang=%{language}'
+
+    def initialize(keyword, language = 'en')
+      @search_url = format(BING_SEARCH_URL, keyword: keyword, language: language)
+    end
+
+    def search
+      uri = URI.parse(@search_url)
+
+      response = Faraday.get(uri)
+
+      response.body if response.status == 200
+    rescue Faraday::ConnectionFailed
+      raise
```

Comment:
The best practice is to raise a custom error. See https://github.com/nimblehq/mia-web/blob/development/app/lib/mia/errors.rb for reference.

Also it would be good to add the same logic to https://github.com/nimblehq/ic-rails-huey-toby/blob/develop/app/services/search/google_search_service.rb to ensure the codebase's consistency